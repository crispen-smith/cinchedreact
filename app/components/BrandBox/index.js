/**
 *
 * BrandBox
 *
 */

import React from 'react';
import styled from 'styled-components';
import MegaLogo from '../MegaLogo';

const BrandBoxBase = styled.div`
  padding: 0;
  margin: 0;
  flex: 1;
`;

function BrandBox() {
  return (
    <BrandBoxBase>
      <MegaLogo />
    </BrandBoxBase>
  );
}

BrandBox.propTypes = {};

export default BrandBox;
