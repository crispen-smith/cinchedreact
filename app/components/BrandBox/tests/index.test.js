import React from 'react';
import renderer from 'react-test-renderer';

import BrandBox from '../index';

describe('<BrandBox />', () => {
  it('Renders a BrandBox', () => {
    const tree = renderer.create(<BrandBox />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
