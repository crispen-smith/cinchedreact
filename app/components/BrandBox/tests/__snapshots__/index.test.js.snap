// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BrandBox /> Renders a BrandBox 1`] = `
<div
  className="BrandBox__BrandBoxBase-bOpzwz dtwMcz"
>
  <div
    className="MegaLogo__MegaLogoBase-gVmnHe fjUeHC"
  >
    <span
      className="MegaLogo__C-hBLyQD CnhnX"
    >
      C
    </span>
    <span
      className="MegaLogo__C-hBLyQD iPmZWc"
    >
      T
    </span>
  </div>
</div>
`;
